lib/sidekiq_adhoc_job/web/job_presenter.rb

```diff
@@ -42,7 +51,8 @@ def self.convert_klass_name_to_presenter(path_name, klass_name)
         queue = SidekiqAdhocJob.config.strategy.get_queue_name(klass_name)
         class_inspector = SidekiqAdhocJob::Utils::ClassInspector.new(klass_name)
         args = class_inspector.parameters(:perform)
-        new(klass_name, path_name, queue, args)
+        require_confirm = SidekiqAdhocJob.config.require_confirm.include?(klass_name.to_s)
```

Comment:
I suggest abstracting this within the Configuration class so we can easily change how we want gem users to configure it without too much code change elsewhere, eg. instead of an allowlist, we could also do a blocklist.

```suggestion
        require_confirm = SidekiqAdhocJob.config.require_confirmation?(klass_name.to_s)
```